src/Gnuplot.jl

```diff
@@ -1140,9 +1140,9 @@ function driver(_args...; is3d=false)
         end
     end
 
-    (doDump)  &&  (execall(gp))
+    # (doDump)  &&  (execall(gp))
```

Comment:
This would prevent the package to update the windows until a `@gp` command (with no arguments!) is given.
Also it would not harm the show machinery, so it should be maintained.

Reply:
I mean, yea? That was the whole point. The idea is that the plot should not be shown unless the users have asked for it to be shown. In the repl this would be if you dont append a semicolon, and in e.g. IJulia this would be equivalent of having the plot be the argument that is returned from the cell. This is how all Julia code behave so I thought it made sense to do the same here.

Reply:
You're right as long as the REPL is concerned.
But we need to distinguish the update / non-update situations also when the code is included in a function.

The trailing `:-` symbol seemed to me a good option since it resembles the hyphenation in natural language...